### PR TITLE
Export to CSV

### PR DESCRIPTION
**Pull request by gaborszita**
Users are now able to export scout reports for a competition to a CSV.

### CODE REVIEW

**Comment by gaborszita** [comment]
@alaninnovates pls review this

File: app/src/screens/data-flow/DataMain.tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ import CompetitionsView from '../competitions-flow/CompetitionsView';
 import UserManager from './UserManager';
 import FormCreationMain from "../form-creation-flow/FormCreationMain";
 import FormCreation from "../form-creation-flow/FormCreation";
+import ExportToCsv from "../export-to-csv-flow/ExportToCsv";
```
Comment (alaninnovates) [inline app/src/screens/data-flow/DataMain.tsx:15]: I think the file should be named `ExportToCSV`, as "CSV" is an abbreviation

File: app/src/screens/export-to-csv-flow/ExportToCsv.js [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+  const exportToCsv = async comp => {
+    setLoading(true);
+    let form;
+    try {
+      form = await FormsDB.getForm(comp.formId);
+    } catch (error) {
+      console.error(error);
+      setLoading(false);
+      Alert.alert('Error', 'An error occurred while fetching the form');
```
Comment (alaninnovates) [inline app/src/screens/export-to-csv-flow/ExportToCsv.js:57]: As a general thing for all alerts, we should give some info on how to resolve it. Maybe something along the lines of "An error occurred when fetching the form. Try reloading the app"
(this goes to all alerts, everywhere)

Comment (gaborszita) [reply app/src/screens/export-to-csv-flow/ExportToCsv.js:57]: 👍

File: app/src/screens/data-flow/DataMain.tsx (hunk 1 above)
Comment (gaborszita) [reply app/src/screens/data-flow/DataMain.tsx:15]: 👍

File: app/src/screens/export-to-csv-flow/ExportToCsv.js [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+          </Text>
+          <Text
+            style={{
+              color: colors.text,
+              marginBottom: 15,
+            }}>
+            Choose a competition to export the scout reports to a CSV file
+          </Text>
+          <ScrollView>
+            {competitionList.map((comp, index) => (
```
Comment (alaninnovates) [inline app/src/screens/export-to-csv-flow/ExportToCsv.js:211]: If there are no competitions, we should display a message

**Pull request merged**